flutter/lib/src/sentry_flutter_measurement.dart

```diff
@@ -0,0 +1,23 @@
+import '../sentry_flutter.dart';
+
+extension SentryFlutterMeasurement on SentryMeasurement {
+  /// Duration of the time to initial display in milliseconds
+  static SentryMeasurement timeToInitialDisplay(Duration duration) {
```

Comment:
you can create new constructors for both ttid and ttfd in `SentryMeasurement` without the need of extensions
if you want to keep the assertion on the duration you can even create a `factory` constructor (still in `SentryMeasurement`)

Reply:
I thought it'd be nice to separate it since `SentryMeasurement` is in the `dart` package and ttid/ttfd don't exist in only dart

Reply:
we already have frames stuff in `SentryMeasurement`, that are not used in pure dart apps, so it's fine to put everything in one place